packages/compass-data-modeling/src/components/new-diagram-form.tsx

```diff
@@ -112,20 +109,27 @@ const FormStepContainer: React.FunctionComponent<{
   );
 };
 
-const SelectListStyles = css({
-  height: 300,
+const selectListStyles = css({
+  maxHeight: 200,
   overflow: 'scroll',
 });
 
 function SelectCollectionsStep({
   collections,
   selectedCollections,
+  automaticallyInferRelationships,
   onCollectionsSelect,
+  onAutomaticallyInferRelationshipsToggle,
 }: {
   collections: string[];
   selectedCollections: string[];
+  automaticallyInferRelationships: boolean;
   onCollectionsSelect: (colls: string[]) => void;
+  onAutomaticallyInferRelationshipsToggle: (newVal: boolean) => void;
 }) {
+  const showAutoInferOption = usePreference(
```

Comment:
Basic Compass FF question, can this be removed now that the FF is enabled?

Reply:
I don't think this one really need to stay for that long, I really wanted it only for the initial algo merging while there was no form checkbox, I'll clean this up in a week or so.